qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -942,50 +961,63 @@ def test_help(self):
+    """
+    def test_reading_conf(self):
+        editorval = self.get_cephfs_shell_cmd_output('set editor ?, set editor')
+        editorval = editorval.replace("'", "", 2)
+        editorval = editorval.split(': ')[1]
+        neweditorval = '???'
+
+        tempconfpath = self.mount_a.client_remote.mktemp(
+            suffix='cephfs-shell.conf')
+        confcontents = "[cephfs-shell]\neditor =  " + neweditorval
+        sudo_write_file(self.mount_a.client_remote, tempconfpath, confcontents)
+
+        finaleditorval = self.get_cephfs_shell_cmd_output(
+            cmd='set editor', shell_conf_path=tempconfpath)
+        finaleditorval = finaleditorval.split(': ')[1]
+        finaleditorval = finaleditorval.replace("'", "", 2)
+
+        self.assertNotEqual(editorval, finaleditorval)
 
-    def test_reading_conf_after_setting_opt_twice(self):
+    def test_reading_conf_with_dup_opt(self):
         """
         Read conf without duplicate sections/options.
         """
-        debugval = self.fs.mon_manager.raw_cluster_cmd('config', 'get',
-                                                       'client','debug_shell')
-        debugval = str_to_bool(debugval)
-
-        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'client',
-                                            'debug_shell', str(not debugval))
-        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'client',
-                                            'debug_shell', str(not debugval))
-        output = self.get_cephfs_shell_cmd_output('set debug')
-        new_debug_val = \
-            str_to_bool(output[output.find('debug: ') + len('debug: ') : ])
-        assert not debugval == new_debug_val
-
-    def test_reading_conf_after_resetting_opt(self):
-        debugval = self.fs.mon_manager.raw_cluster_cmd('config', 'get',
-                                                       'client','debug_shell')
-        debugval = str_to_bool(debugval)
-
-        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'client',
-                                            'debug_shell', str(not debugval))
-        self.fs.mon_manager.raw_cluster_cmd('config', 'rm', 'client',
-                                            'debug_shell')
-        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'client',
-                                            'debug_shell', str(not debugval))
-        output = self.get_cephfs_shell_cmd_output('set debug')
-        new_debug_val = \
-            str_to_bool(output[output.find('debug: ') + len('debug: ') : ])
-        assert not debugval == new_debug_val
+        editorval = self.get_cephfs_shell_cmd_output('set editor')
+        editorval = editorval.replace("'", "", 2)
+        editorval = editorval.split(': ')[1]
+        neweditorval = '???'
+
+        tempconfpath = self.mount_a.client_remote.mktemp(
+            suffix='cephfs-shell.conf')
+        confcontents = "[cephfs-shell]\neditor = ???\neditor = %s\n" % (editorval)
```

Comment:
i don't mind but i would like to keep it current way. it's more compact and cleaner since it's pretty short; above style violates indentation which i really don't like for strings as short as that.

Reply:
I would prefer readability with multiple line over style violation.